### ISSUE TITLE
Umbraco generating appsettings-schema.json in test project

### ISSUE THREAD

**Issue by deMD** [umbraco/Umbraco-CMS#13795]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.4.0

### Bug summary

Since Umbraco 10 a appsettings-schema.json file is copies to the root of the project. However, this file is also generated for unit test projects that references the main Umbraco project. It seems the file is also copies to the root of any project that has in implicit reference to the Umbraco.Cms package.

### Steps to reproduce

- Create a new Umbraco project. 
- Create a Unit test project
- Reference the Umbraco project in the Unit Test project
- Build

### Expected result / actual result

An appsettings-schema.json should only be created in the project that directly references Umbraco.Cms.

**Comment by kjac** [comment]
Hi @deMD,

I can see this might be a little annoying. I believe the `Umbraco.Cms` project copies the schema file to the build output folder, which in your case is the unit test output directory.

The schema file is meant as an aid to devs when configuring Umbraco. The most common use case for this of course is referencing `Umbraco.Cms` from a web project, but there might be other use cases as well, which we don't want to break.

If we look at the project source code, it really is just a container for referencing in other projects and for holding the schema file:

![image](https://user-images.githubusercontent.com/7405322/217456074-68f77594-509f-4d6a-99db-5038286e161e.png)

In other words, you should not be referencing `Umbraco.Cms` into your unit tests, but instead take dependencies on the NuGet packages your tests need to run - i.e. [`Umbraco.Cms.Core`](https://www.nuget.org/packages/Umbraco.Cms.Core/), [`Umbraco.Cms.Infrastructure`](https://www.nuget.org/packages/Umbraco.Cms.Infrastructure/) etc. etc. ... this in turn will likely also speed up your build time, as you no longer have to bring the entire CMS into your unit tests 😄
